frb_example/with_flutter/lib/bridge_generated.io.dart

```diff
@@ -0,0 +1,417 @@
+// AUTO GENERATED FILE, DO NOT EDIT.
+// Generated by `flutter_rust_bridge`.
+
+// ignore_for_file: non_constant_identifier_names, unused_element, duplicate_ignore, directives_ordering, curly_braces_in_flow_control_structures, unnecessary_lambdas, slash_for_doc_comments, prefer_const_literals_to_create_immutables, implicit_dynamic_list_literal, duplicate_import, unused_import, prefer_single_quotes, prefer_const_constructors, use_super_parameters, always_use_package_imports, annotate_overrides, invalid_use_of_protected_member
+
+import "bridge_definitions.dart";
+import 'dart:convert';
+import 'dart:typed_data';
+import 'dart:async';
+import 'package:flutter_rust_bridge/flutter_rust_bridge.dart';
+import "bridge_generated.dart";
+import 'dart:ffi' as ffi;
+
+extension FlutterRustBridgeExampleImplIoExt on FlutterRustBridgeExampleImpl {
+// Section: api2wire
+
+  ffi.Pointer<wire_uint_8_list> _api2wire_String(String raw) {
```

Comment:
I wonder whether this work? Functions starting w/ "_" will only be visible in current file, or other files that is connected via `part/part of`.

So, maybe need to use `part`

Reply:
`part` does not allow branching imports, which is why I'm avoiding using it for now. I'll see if I can make use of it later.

Reply:
I see. So does all those api2wire work? Especially I see private methods in all 3 files. For example, `_wire2api_uint_8_list` in bridge_generated.dart.

Since we generate a ton of functions, we definitely do not want to expose it publicly to confuse the IDE auto completer etc...